src/jackdaw/streams/xform.clj

```diff
@@ -0,0 +1,55 @@
+(ns jackdaw.streams.xform
+  "Helper functions for working with transducers."
+  (:gen-class)
+  (:require [jackdaw.serdes :as js]
+            [jackdaw.streams :as j])
+  (:import org.apache.kafka.streams.kstream.Transformer
+           org.apache.kafka.streams.KeyValue
+           [org.apache.kafka.streams.state KeyValueStore Stores]
+           org.apache.kafka.streams.StreamsBuilder))
+
+(defn kv-store-swap-fn
```

Comment:
from the `-fn` suffix I would have thought it would return a function, but it's actually doing the swapping from what I understand.
Maybe something like `default-kv-store-swap` would be more clear?
Do we need the `-swap!` as well if it's a write?

Reply:
> Maybe something like `default-kv-store-swap` would be more clear?

I like this!


> Do we need the `-swap!` as well if it's a write?

According to the [style guide](https://github.com/bbatsov/clojure-style-guide#changing-state-fns-with-exclamation-mark):
"The names of functions/macros that are not safe in STM transactions should end with an exclamation mark (e.g. reset!)." 


So yeah... I can get behind that.

Reply:
oh wait, is this the reducing function for the state store?